### ISSUE TITLE
Unable to select new solute with VO on mobile

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molarity#158]
For https://github.com/phetsims/QA/issues/436.
When using VO on iPad OS, you are able to select the solute option box and scroll through options, but double clicking on your option just reads it out again, and does not select it.

**Comment by KatieWoe** [comment]
iOS 12 does not seem to have this trouble.  I did also notice that it was a bit difficult to transition focus from the popup box to the solute options, instead of to the next or previous object.

**Comment by terracoda** [comment]
Thanks @KatieWoe, this was also a problem in my interview on Monday!
We obviously have not instrumented this interaction for mobile yet.

@twant could you check with @jessegreenberg as a first step?
My guess is that we need:
- double tap to open the list box
- swipe up and down to go through the items
- double tap to select and close item with focus

I am not sure of a cancel gesture  - to close without selecting?

**Comment by jessegreenberg** [comment]
I am seeing this with iPadOS VO too. VoiceOver has the correct bounds for the items, but we aren't receiving any events when we try to click items. Im going to test how iOS VoiceOver handles role="options" in another example.

**Comment by jessegreenberg** [comment]
Alright, fixed in the above commit. @KatieWoe thanks for pointing this out! Can you please verify it is fixed?

@terracoda said
>My guess is that we need:

>double tap to open the list box
>swipe up and down to go through the items
>double tap to select and close item with focus
>I am not sure of a cancel gesture - to close without selecting?

Thats almost exactly how it behaves! It behaves just like https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html
 - Double tap to open the list box
 - Swipe right and left to up and down through the items
 - Swipe up and down to read the labels in detail
 - Double tap to select and close item with focus

I am not aware of a cancel gesture either, but selecting the already active item will have this effect.

**Comment by KatieWoe** [comment]
Looks good on iPadOS 13.  Still having trouble on iOS 12 trying to transition focus from the popup box to the solute options, instead of to the next or previous object

**Comment by jessegreenberg** [comment]
Over Slack, @KatieWoe explained that on iOS 12 opening the combo box does not move focus into the list box. And once open, you cannot swipe left/right to move focus into the listbox. The only way for the listbox to work is to tap inside of it once it is open. Then swiping through items works as expected.

@KatieWoe do you see this problem on iOS 12 with the listbox in this example?
https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

**Comment by KatieWoe** [comment]
It doesn't, but it also doesn't close the menu when you make a selection

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe - then it is in some way related to our implementation. So now the question is whether or not it is worth fixing a bug that is not on the latest iPadOS/iOS. Would be good to discuss with @terracoda and @emily-phet.

**Comment by emily-phet** [comment]
@KatieWoe Does PhET focus only on supporting the latest iOS for iPad/app etc?

**Comment by KatieWoe** [comment]
Yes.  However, I was still doing a bit of testing since the upgrade happened recently and the new OS has a few problems being worked out.  I'll leave it up to you if you think any iOS 12 problems need to be addressed, but I figured it was at least good to be aware of them for a bit.

**Comment by jessegreenberg** [comment]
I forgot, I have an old iPad2, ill see if it is happening in iOS 9.

**Comment by jessegreenberg** [comment]
We talked about this in a meeting with @emily-phet and @zepumph - we are going to continue to focus on the latest iOS/Safari with the latest screen reader. This means that we will not fix the iOS12 issue mentioned. But it is good to be aware of, thanks for pointing it out @KatieWoe.

**Comment by terracoda** [comment]
@jessegreenberg, I am fine with the decision to support the latest software, but I just want to verify that it is nota focus management issue in our implementation.

For example are we moving focus correctly from the button to the selected item upon opening of the the list of options?

**Comment by jessegreenberg** [comment]
Yes, definitely worth checking @terracoda. It isn't obvious what the issue is, but we are trying to move focus the way you describe. When opening the menu with another platform we support you can see the focus move from button to list item.

**Comment by terracoda** [comment]
I posted some discussion points about focus on group widgets to in https://github.com/phetsims/scenery/issues/1005

There seems to be some subtleties about "programatic focus" and "selection" and this resource may also be relevant https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_focus_vs_selection

**Comment by lmulhall-phet** [comment]
When testing to check if this issue is fixed, make sure that the solute menu will close normally, e.g. if you click the reset all button, the solute menu should close. (It doesn't in 1.5.0-dev.23.)
